pyproject.toml

```diff
@@ -6,3 +6,41 @@ requires = [
   "wheel",
 ]
 build-backend = "setuptools.build_meta"
+
+[tool.towncrier]
+package = "molecule"
+package_dir = "molecule"
+filename = "CHANGELOG.rst"
+directory = "changelog/"
```

Comment:
We need to decide on the location: some projects have it under `docs/` to keep the project root cleaner. I personally don't care much, just wanted to point out the possibilities.
I'm also planning to use this value for a GitHub Action checking the presence of change fragments in PRs.

Reply:
I just followed prior art of pytest, so will just leave it as is.